core/streamdata.go

```diff
@@ -29,15 +29,17 @@ type DetectionConfig struct {
 }
 
 type StreamParameters struct {
-	ManifestID   ManifestID
-	RtmpKey      string
-	Profiles     []ffmpeg.VideoProfile
-	Resolution   string
-	Format       ffmpeg.Format
-	OS           drivers.OSSession
-	RecordOS     drivers.OSSession
-	Capabilities *Capabilities
-	Detection    DetectionConfig
+	ManifestID       ManifestID
+	ExternalStreamID string
+	SessionID        string
```

Comment:
It is not being sent to RabbitMQ?

Reply:
It's actually not. We don't need it for the `analyzer` logic and we don't have it from a couple places where we send events (`mist-api-connector` and some events on the `api`), so I didn't want to include it in the "base event fields" for now. When we decide to include it in the future, we can work on those places to have the consistent session ID all around. I was planning of leaving it in the `authWebhook` anyway since it doesn't hurt and will facilitate that change in the future, but let me know if you rather remove it.

Reply:
If we're relatively sure that this field will see use within a few months, then makes sense to keep it in. Else I'm inclined to remove it right now (maybe keeping it in a branch) and re-add later.

Reply:
I think we'll very soon want to differentiate sessions in the stream health dashboard, so I think we will be using it within a few months indeed. Keeping it then 🙆‍♂️